### PR TITLE
demo button spacing fix

### LINKED ISSUE #104: Nit with spacing on demo button
I think it needs to be outside the #audioplayer div. Want to take this one @Ab1d? :smiley:

### PR DESCRIPTION
**Pull request by Ab1d**
closes #104

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -34,15 +35,15 @@ header, footer {
...
-#playerControls {
-    display: none;
-}
-
 #zen-video-title #demo {
     margin-top: 5px;
     font-weight: bold;
     word-wrap: break-word;
 }
+#demo {
```
Comment (shakeelmohamed) [inline css/styles.css]: Where did these numbers come from?
I'd like to see if we can do this without these seemingly random numbers

**Comment by shakeelmohamed** [comment]
Looks good, I'd like to have the screenshot include the tweet button

**Comment by shakeelmohamed** [comment]
Related to #73, there's now not enough space after the error :smiley: 

To reproduce the issue, type `test` in the text box and click the search button

capturfiles_1

**Comment by shakeelmohamed** [comment]
Looks good, let's just add a new CSS rule and I think this should be done!

```css
#zen-video-error {
    text-align: center;
    margin-top: 5%;
}
```

Before that change:

capturfiles_4

After that change:

capturfiles_5

File: css/styles.css [hunk 2]
```diff
@@ -34,15 +35,19 @@ header, footer {
...
-}
-
 #zen-video-title #demo {
     margin-top: 5px;
     font-weight: bold;
     word-wrap: break-word;
 }
+#zen-video-error {
+    text-align: center;
+    margin-top: 5%;
```
Comment (shakeelmohamed) [inline css/styles.css]: Sorry, let's change this to 30px actually. We're not really using percents for margins, my mistake

**Comment by Ab1d** [comment]
fixed, please check now.

**Comment by shakeelmohamed** [comment]
@Ab1d did you push? :smile:

File: css/styles.css (hunk 2 above)
Comment (Ab1d) [inline css/styles.css]: 30px looks like more space, 20px looks better, what say?

30px: 
screen shot 2016-01-03 at 11 42 02 am

20px:
screen shot 2016-01-03 at 11 41 41 am

Comment (shakeelmohamed) [inline css/styles.css]: Sure, let's do 20px! 5% on my browser was 23px, but a round number is better

**Comment by shakeelmohamed** [comment]
Awesome! Thanks for bearing with me @Ab1d

**Comment by Ab1d** [comment]
:) no problem, lets make this thing great.

**Comment by shakeelmohamed** [comment]
Love the attitude! :+1:

**Pull request merged**